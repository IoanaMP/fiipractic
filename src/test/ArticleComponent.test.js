import React from "react";
import { shallow, mount } from "enzyme";
import toJson from "enzyme-to-json";
import AricleComponent from "../components/ArticleComponent";

describe("Articles page test", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<AricleComponent title="prim" />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it("should render title", () => {
    const wrapper = shallow(<AricleComponent title={"ceva"} />);
    expect(wrapper.find(".articleTitle").text()).toBe("ceva");
  });
  it("should mount Article Component", async () => {
    const wrapper = await mount(<AricleComponent title={"ceva"} />);
    expect(wrapper.find(".articleTitle").text()).toBe("ceva");
  });
});
