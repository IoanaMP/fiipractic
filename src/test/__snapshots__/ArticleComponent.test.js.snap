// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Articles page test should match snapshot 1`] = `
<div
  className="articleContainer"
>
  <div
    className="articleTitle"
  >
    prim
  </div>
  <div
    className="articleContent"
  />
</div>
`;
